app/components/developer_query_component.html.erb

```diff
@@ -5,22 +5,31 @@
 
     <div class="flex items-center justify-between py-6">
       <%= form_with url: developers_path, method: :get, class: "flex items-center justify-between sm:w-full" do |form| %>
-        <div data-controller="toggle" data-toggle-close-class="hidden" class="relative z-10 inline-block text-left" id="sort">
-          <div>
-            <button type="button" data-action="toggle#toggle" class="group inline-flex justify-center text-sm font-medium text-gray-700 hover:text-gray-900" aria-expanded="false" aria-haspopup="true">
-              <%= t(".sort.title") %>
-              <%= inline_svg_tag "icons/solid/chevron_down.svg", class: "flex-shrink-0 -mr-1 ml-1 h-5 w-5 text-gray-400 group-hover:text-gray-500" %>
-            </button>
+
+        <% if Feature.enabled?(:filter_by_search_query) %>
```

Comment:
Is it possible to not have an `else` statement on the feature flag? Even if that means moving Sort all the way to the right even before this feature is enabled.

Reply:
Yeah the Sort button will be moved to the right side if we don't add the `else` here. So do you suggest to remove the else statement?

Reply:
Fine by me! The search will eventually be ready for everyone, eventually.